conftest.py

```diff
@@ -1,5 +1,12 @@
-# List of files that pytest should ignore
+# Use legacy numpy printing. This fix is made to keep doctests functional.
+# For more info, see https://github.com/scikit-image/scikit-image/pull/2935 .
+# TODO: remove this workaround once minimal required numpy is set to 1.14.0
```

Comment:
Please add this to `TODO.txt`, then we're ready to go.

Reply:
@stefanv Done!